versions/3.0.4.md

```diff
@@ -3076,7 +3073,7 @@ However, the exact nature of such conversions are implementation-defined.
 
 ##### Examples
 
-For these examples, assume all schemas are in the entry OpenAPI document; for handling of `discriminator` in referenced documents see [Resolving Implicit Connections](#resolving-implicit-connections).
+For these examples, assume all schemas are in a single-document OpenAPI Description; for handling of `discriminator` in referenced documents see [Resolving Implicit Connections](#resolving-implicit-connections).
```

Comment:
How about
```suggestion
For these examples, assume all schemas are in the [entry document](#openapi-description-structure); for handling of `discriminator` in referenced documents see [Resolving Implicit Connections](#resolving-implicit-connections).
```

Reply:
That's fine, I guess it's less words than what I had :-)